siriuspy/siriuspy/psdiag/csdev.py

```diff
@@ -12,15 +12,15 @@ class ETypes(_csdev.ETypes):
         'Current-(SP|Mon) differ', 'Interlocks', 'Wfm error')
 
 
-_et = ETypes  # syntactic sugar
```

Comment:
I had corrected this typo in the previous PR. You are reverting it?

Reply:
That was a merge error, I will fix that!